[MXNET-503] Website landing page for MMS

aaronmarkham opened the pull request:
## Description ##
This is the content for a website landing page for MMS. Navigation updates will come later.

sandeep-krishnamurthy commented on docs/mms/index.md:
```diff
@@ -0,0 +1,91 @@
+# Model Server for Apache MXNet (incubating)
+
+Model Server for Apache MXNet (incubating), otherwise known as MXNet Model Server (MMS), is an open source project aimed at providing a simple yet scalable solution for model inference. It is a set of command line tools for packaging model archives and serving them. The tools are written in Python, and have been extended to support containers for easy deployment and scaling. MMS also supports bas ...
```
Should we say MMS supports both Gluon and MXNet symbolic models along with ONNX models?

sandeep-krishnamurthy commented on docs/mms/index.md:
```diff
@@ -0,0 +1,91 @@
+
+While the name implies that MMS is just for MXNet, it is in fact much more flexible, as it can support models in the [ONNX](https://onnx.ai) format. This means that models created and trained in PyTorch, Caffe2, or other ONNX-supporting frameworks can be served with MMS.
+
+To find out more about MXNet's support for ONNX models and using ONNX with MMS, refer to the following resources:
```
There are some tutorials on exporting PyTorch models to ONNX, should we link it from the work PyTorch models?

sandeep-krishnamurthy commented on docs/mms/index.md:
````diff
@@ -0,0 +1,91 @@
+mxnet-model-server \
+  --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
+```
+
````
Should we show an example output here for clarity of URL, port number etc. that we use in next section? Something like - Your server should start up like below etc.

aaronmarkham replied:
added output sample

sandeep-krishnamurthy commented on docs/mms/index.md:
```diff
@@ -0,0 +1,91 @@
+* [MXNet Forum - MMS Discussions](https://discuss.mxnet.io/c/mxnet-model-server)
+
+
+## Further Reading
```
Probably we should also reference AWS blogpost on using MMS with Fargate for productionization of DL model?

aaronmarkham replied:
added link to blog

szha commented:
Pinging @piiswrong for review

lupesko commented on docs/mms/index.md:
```diff
@@ -0,0 +1,107 @@
+* [Create an ONNX model (that will run with MMS) from PyTorch](https://github.com/onnx/onnx-mxnet/blob/master/README.md#quick-start)
+
+
+## Using Containers
```
Should we point users to the Docker Hub builds?
https://hub.docker.com/r/awsdeeplearningteam/mms_cpu and https://hub.docker.com/r/awsdeeplearningteam/mms_gpu

aaronmarkham replied:
I added a docker pull command and a link to the parent on docker hub. I think it would be nicer to have an actual organization page and have the repos listed there. One stop shopping, but this should be fine for now!

aaronmarkham commented:
Thanks @john-andrilla - I've addressed each comment in my latest commit (fixed stuff, removed "free").
How about "a flexible, scalable, multi-framework serving solution"?
